oracle/oracle/rewards/controller.py

```diff
@@ -43,6 +43,22 @@
 w3 = Web3()
 
 
+class WithdrawalsCache:
+    block: BlockNumber
+    withdrawals: Wei
+
+    def __init__(self, block: BlockNumber, withdrawals: Wei):
```

Comment:
Params are optional. It's better to annotate them as Optional, or pass None as default

```
def __init__(self, block: BlockNumber = None, withdrawals: Wei = None)
```